src/components/title-list-item/title-list-item.js

```diff
@@ -7,86 +7,116 @@ import { Headline } from '@folio/stripes/components';
 import shouldFocus from '../should-focus';
 import styles from './title-list-item.css';
 import InternalLink from '../internal-link';
+import IdentifiersList from '../identifiers-list';
+import ContributorsList from '../contributors-list';
 
 const cx = classNames.bind(styles);
+const CONTRIBUTORS_LIMIT = 3;
 
 function TitleListItem({
   item,
   link,
   active,
   showSelected,
   showPublisherAndType,
+  showContributors,
+  showIdentifiers,
   onClick,
   headingLevel
 }) {
-  return !item ? (
-    <div
-      className={cx('skeleton', {
-        'is-selected-visible': showSelected,
-        'is-publisher-and-type-visible': showPublisherAndType
-      })}
-    />
-  ) : (
-    <InternalLink
-      data-test-eholdings-title-list-item
-      to={link}
-      className={cx('item', {
-        'is-selected': active
-      })}
-      onClick={(e) => {
-        if (onClick) {
-          e.preventDefault();
-          onClick();
-        }
-      }}
-    >
-      <Headline
-        data-test-eholdings-title-list-item-title-name
-        margin="none"
-        size="medium"
-        tag={headingLevel || 'h3'}
+  const hasContributors = item && item.contributors && item.contributors.length > 0 && showContributors;
```

Comment:
I think it could be shorter
```suggestion
  const hasContributors = item?.contributors?.length > 0 && showContributors;
```